line-bot-api-client/build.gradle

```diff
@@ -13,13 +13,16 @@
  * License for the specific language governing permissions and limitations
  * under the License.
  */
+apply plugin: 'java-library'
```

Comment:
How about replace https://github.com/line/line-bot-sdk-java/pull/446/files/3a8efdb046303f62ee575d1329be7c70497c971e#diff-c197962302397baf3a4cc36463dce5eaR79 this line?

Reply:
link miss. here

https://github.com/line/line-bot-sdk-java/blob/3a8efdb046303f62ee575d1329be7c70497c971e/build.gradle#L79

Reply:
ok. I'll do this